design/cstor/v1/cstorvolume.md

```diff
@@ -0,0 +1,351 @@
+# Proposed v1 CStorVolumes and CStorVolumeReplica Schema
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Goals](#goals)
+* [Schema Proposal](#volume-schema-proposal)
+
+## Introduction
+
+This proposal highlights the enhancements and migration of current `CStorVolumes` and `CStorVolumeReplicas` schema and
+proposes improvements.
+
+# Goals
+
+The major goal of this document is to freeze the APIs schema for `CStorVolumes` and `CStorVolumeReplica`.
+Apart from this the document focuses on following aspects too:
```

Comment:
Remove these links. the only goal is to propose v1 schema for `CStorVolumes` and `CStorVolumeReplica`

Reply:
done